### DISCUSSION TITLE
如何自定义url

### DISCUSSION THREAD

**Discussion by zuoer96** [vuejs/vitepress#2959] [Q&A]
**假设本地目录：**
docs
|- 01_文章1.md
|- 02_文章2.md

**访问时的url为：**
docs/01_文章1.md
docs/02_文章2.md

**可以将访问的url调整为：**
docs/post1.html
docs/post2.html
吗？

在本地写的时候，文件名需要用中文
但是线上 01_文章1 这种的url不太美观

**Comment by brc-dd** [comment]
https://vitepress.dev/guide/routing#route-rewrites
